plugins/transformer/basic/docs/overview.md

```diff
@@ -58,3 +59,11 @@ You can also use the `obfuscate_sensitive_columns` transformation to automatical
 
 Note: transformations are applied sequentially. If you rename tables, the table matcher configuration of subsequent transformations will need to be updated to the new names.
 Note: escape syntax is [SJSON sytax](https://github.com/tidwall/sjson?tab=readme-ov-file#path-syntax).
+
+
+
+
+Edge Cases and limitations for `drop_rows` transformation:
+- The `drop_rows` transformation only supports non-list columns
```

Comment:
```suggestion
- Only non-list columns are supported
```

We don't need to repeat `drop_rows`, we're already in the context of `drop_rows` limitations